shh/src/Shh.hs

```diff
@@ -53,6 +53,7 @@ module Shh
     , captureEndBy
     , captureEndBy0
     , captureLines
+    , captureWords
```

Comment:
It feels weird having to import Shh.Internal just to get this function, so exposing it from the "main" module makes sense to me.